dns_test.go

```diff
@@ -374,17 +374,47 @@ func TestDNSProtocol(t *testing.T) {
 
 		_, port, _ := net.SplitHostPort(addr.String())
 
-		// Force IPv4
+		// Default DNS Server port
```

Comment:
Aha.. Fixed! Maybe a separate Test func could be better? Please feel free to let me know.